css/src/atomics/README.md

```diff
@@ -0,0 +1,54 @@
+# Atlas Atomics
+
+Welcome to Atomics ⚛! These are small, single purpose, css classes that can be used to compose great things. Visit this article for a [longer definition of Atomic CSS here](https://css-tricks.com/lets-define-exactly-atomic-css/#:~:text=Atomic%20CSS%20is%20the%20approach,names%20based%20on%20visual%20function).
```

Comment:
Seems like this shorter link works as well https://css-tricks.com/lets-define-exactly-atomic-css/

Reply:
"CSS classes"?